app/src/main/res/layout/fragment_house_detail.xml

```diff
@@ -19,14 +20,27 @@
             android:visibility="gone"
             android:layout_gravity="center"/>
 
+        <TextView
+            android:id="@+id/house_detail_fragment_error"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="@android:color/white"
+            style="@style/style_error_message_text_view"
+            android:visibility="gone"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"/>
+
         <ImageView
             android:id="@+id/house_detail_fragment_image"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            android:layout_marginTop="@dimen/house_detail_fragment_view_margin_top"/>
+            android:layout_marginTop="@dimen/house_detail_fragment_view_margin_top"
+            tools:ignore="ContentDescription" />
```

Comment:
add the content description instead of the ignore

Reply:
Done!